src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -232,7 +232,7 @@ public class CoinstakeContext
         private readonly IWalletManager walletManager;
 
         /// <summary>Provides value for PoS reward and checks PoS kernel.</summary>
-        private readonly IPosConsensusValidator posConsensusValidator;
+        //private readonly IPosConsensusValidator posConsensusValidator;
```

Comment:
Remove commented code please if it's not needed

Reply:
agreed